browser-test/src/applicant/navigation/northstar_application_navigation_common_intake.test.ts

```diff
@@ -181,22 +178,11 @@ test.describe('Applicant navigation flow', {tag: ['@northstar']}, () => {
         /* wantTrustedIntermediary= */ false,
         /* wantEligiblePrograms= */ [secondProgramName],
       )
-
-      await validateScreenshot(
-        page,
-        'cif-eligible-guest-confirmation-page',
-        /* fullPage= */ true,
-        /* mobileScreenshot= */ true,
-      )
       await validateAccessibility(page)
 
       await page.click('text="Apply to programs"')
-
-      await validateScreenshot(
-        page,
-        'cif-submission-guest-login-prompt-modal',
-        /* fullPage= */ false,
-        /* mobileScreenshot= */ true,
+      expect(await page.innerHTML('h2')).toContain(
```

Comment:
**suggestion:** Let's add a check to see if the login-prompt-modal appears here, so that we're not losing anything by removing the screenshot.